qiskit/providers/backend_compat.py

```diff
@@ -13,185 +13,275 @@
+                name,
+            )
+            unsupported_instructions.append(name)
+
+    for name in unsupported_instructions:
+        all_instructions.remove(name)
+
+    # Create empty inst properties from gate configs
+    for name, spec in gate_configs.items():
+        if hasattr(spec, "coupling_map"):
+            coupling_map = spec.coupling_map
+            prop_name_map[name] = dict.fromkeys(map(tuple, coupling_map))
+        else:
+            prop_name_map[name] = None
+
+    # Populate instruction properties
+    if properties:
+        qubit_properties = [
+            QubitProperties(
+                t1=properties.qubit_property(qubit_idx)["T1"][0],
+                t2=properties.qubit_property(qubit_idx)["T2"][0],
+                frequency=properties.qubit_property(qubit_idx)["frequency"][0],
+            )
+            for qubit_idx in range(0, configuration.num_qubits)
+        ]
+
+        in_data["qubit_properties"] = qubit_properties
 
-    # Standard gates library mapping, multicontrolled gates not included since they're
-    # variable width
-    name_mapping = get_standard_gate_name_mapping()
-    target = None
-    if custom_name_mapping is not None:
-        name_mapping.update(custom_name_mapping)
-    faulty_qubits = set()
-    # Parse from properties if it exsits
-    if properties is not None:
         if filter_faulty:
-            faulty_qubits = set(properties.faulty_qubits())
-        qubit_properties = qubit_props_list_from_props(properties=properties)
-        target = Target(
-            num_qubits=configuration.n_qubits,
-            qubit_properties=qubit_properties,
-            concurrent_measurements=getattr(configuration, "meas_map", None),
-        )
-        # Parse instructions
-        gates: Dict[str, Any] = {}
+            faulty_qubits = properties.faulty_qubits()
+
         for gate in properties.gates:
             name = gate.gate
-            if name in name_mapping:
-                if name not in gates:
-                    gates[name] = {}
-            else:
-                raise QiskitError(
-                    f"Operation name {name} does not have a known mapping. Use "
-                    "custom_name_mapping to map this name to an Operation object"
-                )
-
             qubits = tuple(gate.qubits)
-            if filter_faulty:
-                if any(not properties.is_qubit_operational(qubit) for qubit in qubits):
-                    continue
-                if not properties.is_gate_operational(name, gate.qubits):
-                    continue
+            if name not in all_instructions:
+                logger.info(
+                    "Gate property for instruction %s on qubits %s is found "
+                    "in the BackendProperties payload. However, this gate is not included in the "
+                    "basis_gates or supported_instructions, or maybe the gate model "
```

Comment:
```suggestion
                    "basis_gates, or maybe the gate model "
```

`supported_instructions` is dropped in your implementation right?

Reply:
I removed the entire logging block :)